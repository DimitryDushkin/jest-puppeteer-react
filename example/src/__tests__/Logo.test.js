import React from 'react';
import { mount } from 'enzyme';

import Logo from '../Logo';

describe('Logo', () => {
    test('should render correctly', () => {
        const wrapper = mount(<Logo />);

        expect(wrapper).toMatchSnapshot();
    });
});
