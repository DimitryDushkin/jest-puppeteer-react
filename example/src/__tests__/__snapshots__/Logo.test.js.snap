// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render correctly 1`] = `
<Logo>
  <div>
    test
  </div>
</Logo>
`;
